downstream/modules/aap-hardening/proc-configure-centralized-logging.adoc

```diff
@@ -22,16 +22,16 @@ awx-manage print_settings LOG_AGGREGATOR_MAX_DISK_USAGE_GB
 +
 If this field is not set to the organizationally defined log capture size, then follow the configuration steps.
 
-. Check `LOG_AGGREGATOR_MAX_DISK_USAGE_PATH` field in the {ControllerName} configuration for the log file location to `/var/log/awx`.  On the host, execute:
+. Check `LOG_AGGREGATOR_MAX_DISK_USAGE_PATH` field in the {ControllerName} configuration for the log file location to `/var/lib/awx`.  On the host, execute:
 +
 ----
 awx-manage print_settings LOG_AGGREGATOR_MAX_DISK_USAGE_PATH
 ----
 +
-If this field is not set to `/var/log/awx`, then follow these configuration steps: 
+If this field is not set to `/var/lib/awx`, then follow these configuration steps: 
 +
 --
-.. Open a web browser and navigate to https://<automation controller server>/api/v2/settings/logging/, where <FQDN> is the fully-qualified hostname of your {ControllerName}. If the btn:[Log In]" button is displayed, click it, log in as an {ControllerName} adminstrator account, and continue.
+.. Open a web browser and navigate to https://<automation controller server>/api/v2/settings/logging/, where <automation controller server> is the fully-qualified hostname of your {ControllerName}. If the btn:[Log In]" button is displayed, click it, log in as an {ControllerName} adminstrator account, and continue.
```

Comment:
this change looks good, but i noticed in the rendered doc, the string "https://<automation" gets converted to a URL link.  is there a way to render the text so that it doesn't get converted to a link?  since this is an example URL, it will never work anyway.

Reply:
@jce-redhat I saw that too. I've been playing around with that section since last week and have been unable to keep it from rendering as a link. as long as `https:` is there, nothing works.

Thanks for the review!